src/main/release-scripts/validate-release.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+RELEASE_VERSION=$1
+WORKSPACE=${WORKSPACE:-'.'}
+CHANGELOG=$WORKSPACE/changelog.txt
+README=$WORKSPACE/readme.md
+
+if [ `git tag -l | grep $RELEASE_VERSION` ]
```

Comment:
yes, because otherwise you don't have the tags locally and the check won't fail even if these exist already.